CONTRIBUTING.md

```diff
@@ -227,8 +227,7 @@ And in case we didn't emphasize it enough: **we love tests!**
 
 #### Testing
 
-Testing is done with mocha. You can invoke the tests by running `npm test` or `npm run-script
-test-all`. Both scripts are running `node tests/runner` may be with additional params. To contribute
+Testing is done with mocha. You can invoke the tests by running `npm test` or `npm test:all`. Both scripts are running `node tests/runner` may be with additional params. To contribute
```

Comment:
I think when using `npm` it needs to be `npm run test:all`. Skipping the `run` only works in `yarn`.

Reply:
Yup, you are right, my bad! And both work with `yarn`.